### PR TITLE
DBL token required +  some fixes

### PR DESCRIPTION
**Pull request by lumap**
-The dbl token is now required to start the module
-Removed some code that makes no sense
-Made error messages consistent

### CODE REVIEW

**Comment by tonkku107** [comment]
- Token isn't necessary for running the webhook. Making the token required would also be a breaking change.
- The code that "made no sense" that you removed was functionality making the id property optional if the client has been passed. (As it can be taken from the client)

File: src/index.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ class DBLAPI extends EventEmitter {
    * @private
    * @returns {Promise<Object>}
    */
-  _request(method, endpoint, data) {
+  async _request(method, endpoint, data) {
```
Comment (tonkku107) [inline src/index.js:82]: This is incorrect as the function returns a Promise instead

**Comment by matthew-st** [comment]
wouldn't making the token required kinda take away some use from the module as i'm sure you could use this module for server webhooks, please do correct me if im wrong though

**Comment by jpbberry** [comment]
This pull request is out of date due to the new rewrite.

**Pull request closed without merging**